hs_access_control/models/user.py

```diff
@@ -1088,7 +1089,8 @@ def owned_resources(self):
         """
 
         if not self.user.is_active:
-            raise PermissionDenied("Requesting user is not active")
+            # An innactive user can still own resources so don't raise PermissionDenied
+            warnings.warn("User is not active")
```

Comment:
related to [4542](https://github.com/hydroshare/hydroshare/pull/4542/files/ac7bbf7ff7e39c5ddefcb042a8e3dab36be45034#r829753014)

Reply:
`user.is_active` is most likely checked in many other places as part of access control. I am not sure what side effects this change would have. Since access control have extensive unit tests, I am hoping it should reveal any side effects.